_includes/copy_banner.html

```diff
@@ -4,22 +4,6 @@
         {% assign primary_title = page.primary_title | default: primary_title | default: '' %}
         {% assign primary_link_url = layout.primary_link_url | default: page.url %}
         {% if primary_title.size > 0 %}<h1><a href="{{ primary_link_url }}">{{primary_title}}</a></h1>{% endif %}
-
-        <div class="search">
```

Comment:
`.search` had `z-index: 2` forcing the search results appear on top of `.search-overlay`; that is lost with the new design.